src/lib.rs

```diff
@@ -799,6 +839,7 @@ impl<T: Clone + Num + MulAdd<Output = T>> MulAdd<Complex<T>> for Complex<T> {
         Complex::new(re, im)
     }
 }
+
```

Comment:
Adding a newline here breaks consistency to other code sections (ref implementation is usually directly below without a newline to the owned one).

Reply:
Sorry, I'm struggling to see what you're referring to w.r.t. implementations on references. Could you point me to a line number where this happens?